qt/python/mantidqt/project/projectsaver.py

```diff
@@ -52,7 +52,7 @@ def save_project(self, file_name, workspace_to_save=None, plots_to_save=None, in
             saved_workspaces = ADS.getObjectNames()
 
         # Generate plots
-        plots_to_save_list = PlotsSaver().save_plots(plots_to_save)
+        plots_to_save_list = PlotsSaver().save_plots(plots_to_save, not project_recovery)
```

Comment:
I'm a bit confused as to the purpose of using not here instead of just passing in project_recovery, as you then use not in plotsaver.py. Was there a reason?

Reply:
For some reason project_recovery's meaning is flipped, I was aiming for this code change to be as low impact as possible, mindful of release soon, so I just flipped the boolean value to work inside the function before passing it into save_plots.